.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,68 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: 'CodeQL'
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 13 * * 6'
```

Comment:
Looks like the schedule is randomised across the repos at the moment. Do you think that makes sense, or would it be better for us to get all the notifications at once?

Reply:
A specific scan day/time make sense 🤔  
What day makes sense? Something like Every Monday at 9am?

Reply:
Sounds good 👍